MarsMission/ViewModels/WeatherDataViewModel.swift

```diff
@@ -35,7 +36,10 @@ class WeatherDataViewModel {
 		DispatchQueue.main.async { [self] in
 			self.weatherData = weatherData
 			self.view.populateWeatherData(weatherData.lastUpdated!,
-										  weatherData.weatherStation!)
+										  weatherData.weatherStation!,
```

Comment:
Why do you have this function if you're using a collection view? Make sure to understand how arrays work